server/server.go

```diff
@@ -34,6 +34,11 @@ func newCribbageServer(dbFactory persistence.DBFactory) *cribbageServer {
 func (cs *cribbageServer) NewRouter() http.Handler {
 	router := gin.Default()
 
+	// health check route
+	router.GET(`/health`, func(c *gin.Context) {
+		c.String(http.StatusOK, `Healthy!`)
+	})
```

Comment:
:crossed_fingers: this doesn't accidentally conflict with existing things in docker compose/non-lambda deploys:#

Reply:
Is there a reason it would? It's unique among the routes we have configured, which are the same routes used in a normal or a lambda deploy

Reply:
Actually it kinda does because of [this issue](https://github.com/joshprzybyszewski/cribbage/issues/99); we try to serve _any_ route at the root level with our static files which is kinda not a good thing :#